docs/index.md

```diff
@@ -1,56 +1,28 @@
 ---
-sidebarDepth: 2
+layout: home
+
+title: VitePress
+titleTemplate: Vite & Vue Powered Static Site Generator
+
+hero:
+  name: VitePress
+  text: Vite & Vue Powered Static Site Generator
+  tagline: Simple, minimal, yet strikingly powerful. Meet the modern SSG framework you've always wanted.
+  actions:
+    - theme: brand
+      text: Get Started
+      link: /guide/getting-started
+    - theme: alt
+      text: View on GitHub
+      link: https://github.com/vuejs/vitepress
+
+features:
+  - title: Vite, The DX that can't be beat
+    details: Feel the speed of Vite. Instant server start and lightning fast HMR that stays fast regardless of the app size.
+  - title: Power of Vue meets Markdown
+    details: Enjoy the outstanding feature set of Vue Component in markdown, and develop custom themes with Vue.
+  - title: Simple and minimal, always
```

Comment:
```suggestion
  - title: Designed to be easy to use with minimum configuration
```

"Simple" and "easy" tend to be subjective as far as whether it is actually simple or easy for users. So in general, I tend to avoid describing anything this way in case people find it complicated (which might make them feel stupid).

By saying this is our design philosophy, this helps to communicate our intent  clearly without making assumptions of our user's abilities or preferences.

Reply:
Nice, I like the idea. Can we make it a bit shorter so that all heading in the box has similar length 🤔 I can't come up with good one now but something like "Designed to be minimum, always" (I don't know I don't know what I'm saying right now).

The issue 👇 

<img width="1068" alt="Screen Shot 2022-05-31 at 22 24 21" src="https://user-images.githubusercontent.com/3753672/171184331-e2cf3de9-3a2b-4437-80e0-c239e991c11d.png">